README.md

```diff
@@ -20,7 +26,14 @@ A Foreman Plugin for TeamDynamix. It manages a host lifecycle as a corresponding
     :password: 'password'
     :create:
       :StatusID: integer_id
-      :OwningCustomerName: string
```

Comment:
IS OwningCustomerName something we need to set usually?

Reply:
no it was just an example and have been generalized in the latest commit